release/community-stable/clearlinux/docker/Dockerfile

```diff
@@ -4,69 +4,18 @@
 # Docker image file that describes an Intel Clear Linux image with PowerShell
 # installed from full PowerShell linux tar.gz package
 
-# Define arg(s) needed for the From statement
-ARG fromTag=base
-ARG imageRepo=clearlinux
-
-FROM ${imageRepo}:${fromTag} AS installer-env
+FROM clearlinux:latest AS installer-env
 
 # Define Args for the needed to add the package
-ARG PS_VERSION=6.1.0
+ARG PS_VERSION=7.0.1
 ARG PS_PACKAGE=powershell-${PS_VERSION}-linux-x64.tar.gz
 ARG PS_PACKAGE_URL=https://github.com/PowerShell/PowerShell/releases/download/v${PS_VERSION}/${PS_PACKAGE}
-ARG PS_INSTALL_VERSION=6
+ARG PS_INSTALL_VERSION=7
 
 # Download the PowerShell Core Linux tar.gz and save it
 ADD ${PS_PACKAGE_URL} /tmp/powershell-linux.tar.gz
```

Comment:
More FYI, This will be in the final image because this layer is committed at this point.